packages/libs/src/sdk/sdk.ts

```diff
@@ -293,7 +293,7 @@ const initializeApis = ({
     services.logger
   )
   const tips = new TipsApi(generatedApiClientConfig)
-  const { resolve } = new ResolveApi(generatedApiClientConfig)
+  const resolve = new ResolveApi(generatedApiClientConfig)
```

Comment:
what's this?

Reply:
https://docs.audius.org/developers/api/resolve

Reply:
cool til